src/components/app.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
-import Header from '../containers/header';
 
+import Header from '../containers/header';
+import { Help } from './help/index';
```

Comment:
It's unnecessary to write such row. By default webpack search file index.js

Use `import { Help } from './help`